lib/schemapack.js

```diff
@@ -207,11 +208,19 @@ class SchemaPack {
       return callback(e);
     }
 
+    collectionJSON = generatedStore.collection.toJSON();
+
+    // this needs to be deleted as if an incorrect(non-semver) version is provided to sdk,
+    // it returns a version object, with major, minor, patch, etc. properties as undefined.
+    // this fails validation against v2.1 collection schema definition.
+    if (!collectionJSON.info.version || !collectionJSON.info.version.raw) {
```

Comment:
why do we need to check for !raw? Why not delete info.version whenever it's present?

Reply:
There is one more bug in SDK. When we apply .toJSON() on the final collection created, it does not convert the version object into plain JSON, instead, it returns a PostmanPropertyVersion object, with all these fields set to undefined.